frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/security-and-network-step/encryption.tsx

```diff
@@ -142,9 +142,24 @@ export const Encryption: React.FC<EncryptionProps> = ({ encryption, kms, dispatc
     encryption.clusterWide || encryption.storageClass,
   );
 
+  React.useEffect(() => {
+    if (isMCG) {
+      setEncryptionChecked(true);
```

Comment:
just initializing the state for encrytion with MCG.
Cluster wide encryption need to be true by default for MCG